CHANGES.md

```diff
@@ -42,12 +42,15 @@ _This is a preview of a pending release and subject to change._
 	<li><a href='https://codice.atlassian.net/browse/DDF-1765'>DDF-1765</a> - As a user, I want to add a layer to the Search UI map from a KML file so that I can visualize my KML data
 	</li>
 	<li><a href='https://codice.atlassian.net/browse/DDF-2039'>DDF-2039</a> - As an integrator, I want to make a single request to the catalog framework to get back both the metacard and the resource.
+	</li>
 </ul>
    
 <h3>Task</h3>
 <ul>
 	<li><a href='https://codice.atlassian.net/browse/DDF-2698'>DDF-2698</a> - Create a catalog:export command
 	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2878'>DDF-2878</a> - Create an API for System Administrator Alerts	
```

Comment:
@ricklarsen, should the `Change Log Updated` checkbox be removed from the PR template?